### PR TITLE
My awesomer pull request!

### PR DESCRIPTION
**Pull request by jgrix**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,3 +7,5 @@ developers Sep 27th
 
 # Mastering Markdown
 https://guides.github.com/features/mastering-markdown/
+
+Hi, my name is Jason and I like rainbows and butterflies!
```
Comment (notsig11) [inline README.md]: What about kittens?

File: jgrix-nepal.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Kathmandu, Nepal
```
Comment (notsig11) [inline jgrix-nepal.md:1]: Wait really?  :V  Have you been?

**Comment by solfatari** [comment]
Conflicts when merging with a branch with 1 file in it... You disgust me.

**Review by notsig11 — CHANGES_REQUESTED** [review]
Kittens, man.

File: jgrix-nepal.md (hunk 2 above)
Comment (jgrix) [reply jgrix-nepal.md:1]: I have not. However looking through previous classes that have done this, someone has and I thought it looked interesting. This way I won't forget.

Comment (notsig11) [reply jgrix-nepal.md:1]: Perfect.

File: README.md (hunk 1 above)
Comment (jgrix) [reply README.md]: Excellent point!

**Review by notsig11 — APPROVED** [review]
Much better!  :goberserk:

**Pull request merged**